firmware/firmware.md

```diff
@@ -9802,7 +9802,8 @@ void loop() {
 
 The library provides the following log handlers:
 
-`SerialLogHandler`
+- `SerialLogHandler`
+- Additional community-supported log handlers can be found further below.
```

Comment:
So, it sounds like you don't have an issue with such a comment there but you don't like the actual list of community loghandlers further below? I'd be happy to create a community section of the docs and then link to it from here. Thoughts? Not sure how easy a new section will be to create but I can look into it.